### DISCUSSION TITLE
part-ii/2019-117/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#131] [Question comments]
Part II 2019 Probability and Measure: Paper 2, Section II, K 

https://questions.tripos.org/part-ii/2019-117/

**Comment by eha1234** [comment]
<details><summary>Spoilers</summary>

(c) I'm sure there are other examples, but $min\{\mu(A) , \mu(B)\}$

(d) For the first inequality, $(I \times X) \cup (J \times X) \subset X \times X$ and inclusion exclusion. For the second inequality, note that $I \times J \subset I \times X$ and also $\subset X \times J$

(e) Try $(\epsilon_1 U_1,...,\epsilon_{d-1} U_{d-1}, \epsilon_1...\epsilon_{d-1} U_d)$
</details>
